### ISSUE TITLE
callback is not a function

### ISSUE THREAD

**Issue by im-fortech** [Matt-B/cucumber-js-selenium-webdriver-example#2]
/features/support/world.js:56
  callback();
  ^

TypeError: callback is not a function
    at new World ( /features/support/world.js:56:3)

Any help ?

**Comment by Matt-B** [comment]
Hey!

Apologies for not replying sooner - didn't see this issue until just now!

This is because of a change introduced in [cucumber-js 0.8.0](https://github.com/cucumber/cucumber-js/releases/tag/v0.8.0) - the world constructor no longer receives a callback from Cucumber, so the current setup will no longer work. 

I've since pinned the dependencies in the master branch so it uses an older version of cucumber which will not show this error. I've also started work on a fix for this issue so I can update the project to use the new version of cucumber, but it might be a while before I get the time to finish it off.

Feel free to wait for that, or submit a PR if you fancy trying to fix it yourself.

Thanks,

Matt
